### PR TITLE
Feature/item categories taka

### PR DESCRIPTION
**Pull request by KIN181**

### CODE REVIEW

File: public/css/app.css
Comment (Ditzler01) [inline public/css/app.css:1]: This should not be included.
Add this to your .env

File: public/js/app.js
Comment (Ditzler01) [inline public/js/app.js:1]: This should not be included.
Add this to your .env

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /node_modules
 /public/hot
 /public/storage
+/public/css
```
Comment (keanbillyfernandez) [inline .gitignore:4]: Gitignore file is okay not to be commited

File: resources/views/auth/register.blade.php [hunk 2]
```diff
@@ -1,74 +1,151 @@
...
+	    font-family: FontAwesome;  /* F.A */
+    }
 
-                            <div class="col-md-6">
-                                <input id="email" type="email" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ old('email') }}" required autocomplete="email">
+    body {
+        margin: 0;
+        padding: 0;
+        background-color: #99ccff;
+    }
```
Comment (keanbillyfernandez) [inline resources/views/auth/register.blade.php:39]: the css can be added in outside css file

File: .gitignore (hunk 1 above)
Comment (keanbillyfernandez) [reply .gitignore:4]: But this is okay now

**Pull request merged**